chromium_src/chrome/browser/ui/webui/settings/shared_settings_localized_strings_provider.cc

```diff
@@ -3,8 +3,34 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "build/build_config.h"
+#include "build/buildflag.h"
 #include "chrome/grit/generated_resources.h"
 
+#if BUILDFLAG(IS_WIN)
+#include "brave/components/brave_vpn/features.h"
```

Comment:
this does however need a brave vpn buildflag

Reply:
done